tests/decode-chdlc-02/test.yaml

```diff
@@ -0,0 +1,38 @@
+requires:
+
+  min-version: 8
+
+pcap: ../decode-chdlc-01/hdlc-http_1tx.pcap
+
+checks:
+
+    - filter:
+        count: 1
+        match:
+            event_type: http
```

Comment:
is this indentation correct yaml? Looks like 4 spaces instead of 2?

Reply:
This was in the original file.

Do you recommend fixing the original in addition to this one?